scrabbler/scrabbler.py

```diff
@@ -103,12 +103,17 @@ def find_best_moves(self, rack, num=5):
             moves = across_moves + down_moves
 
         moves.sort(key=lambda move_: move_.score, reverse=True)
-        for move in moves[0:num]:
-            print(move)
+        # for move in moves[0:num]:
+        #     print(move)
```

Comment:
nit - since this is git, we can always look back at past versions of the code if necessary. Therefore there isn't a need to comment out old code, you can just delete it entirely.